addons/survey/views/survey_templates_statistics.xml

```diff
@@ -3,357 +3,344 @@
-                                <span class="badge badge-warning"><span t-esc="input_summary['skipped']"></span> Skipped</span>
-                            </span>
-                        </h4>
-                        <t t-if="question_with_correct_answers">
-                            <t t-set="correct_answers" t-value="question.get_correct_answer_ids()" />
-                            <div t-if="correct_answers">
-                                <t t-if="question.question_type == 'simple_choice'">
-                                    <p><span>Correct answer</span>: <span class="font-weight-bold" t-esc="correct_answers[0].value"></span></p>
-                                </t>
-                                <t t-if="question.question_type == 'multiple_choice'">
-                                    <span>Correct answers</span>:
-                                    <ul>
-                                        <li t-foreach="correct_answers" t-as="correct_answer">
-                                            <t t-esc="correct_answer.value" />
-                                        </li>
-                                    </ul>
-                                </t>
+
+                <div t-foreach="question_and_page_data" t-as='question_data'>
+                    <t t-set="question" t-value="question_data['question']"/>
+                    <t t-if="question_data['is_page']">
+                        <h2 class="mt16 text-uppercase text-muted" t-field="question.title"/>
+                        <div t-field="question.description" class="oe_no_empty" />
+                        <hr class="mt-2 pb-1 mb-1"/>
+                    </t>
+                    <div t-else="" class="ml-4 mt-4">
+                        <t t-set="question_scoring" t-value="survey.scoring_type in ['scoring_with_answers', 'scoring_without_answers'] and question.question_type in ['simple_choice', 'multiple_choice']"/>
+                        <t t-set="comment_lines" t-value="question_data['comment_line_ids']"/>
+                        <t t-set="graph_data" t-value="question_data['graph_data']"/>
+                        <t t-set="table_data" t-value="question_data['table_data']"/>
+
+                        <div class="d-flex align-items-start mb-2">
+                            <div class="mr-auto">
+                                <h5 t-field="question.title" class="mb-1"/>
+                                <div t-if="question_scoring">
+                                    <t t-if="len(question_data['right_answers']) == 1">
+                                        <span>Right answer: </span><strong t-esc="question_data['right_answers'][0]['value']"/>
+                                    </t>
+                                    <t t-elif="len(question_data['right_answers']) > 1">
+                                        <span>Right answers: </span><strong>
```

Comment:
Couldn't you just t-if the span to add the plural form but loop all the time ?

Reply:
A t-if to add an "s" ! :vertical_traffic_light:

Reply:
🚩